torch/distributions/kl.py

```diff
@@ -411,7 +411,7 @@ def _kl_pareto_pareto(p, q):
     t1 = q.alpha * scale_ratio.log()
     t2 = -alpha_ratio.log()
     result = t1 + t2 + alpha_ratio - 1
-    result[p.support.lower_bound < q.support.lower_bound] = inf
+    result[(p.support.lower_bound < q.support.lower_bound).byte()] = inf
```

Comment:
Note "masked functions do not accept bool yet"

Reply:
Masked functions and indexing are supported for bool masks.
One of the previous PRs in this stack fixed this issue - indexing on scalars.